docs/Guides/Scim/Guide-to-sync-users-in-vendasta.md

```diff
@@ -0,0 +1,42 @@
+# Guide to use Vendasta SCIM APIs to sync Users
+
+
+## Overview
+System for Cross-domain Identity Management ([SCIM](https://en.wikipedia.org/wiki/System_for_Cross-domain_Identity_Management)) focuses on syncing user accounts and permissions between systems but has an extension system that allows syncing any type of record. There are several well known endpoint paths for discovering what the server supports and resource schema definitions.
+
+
+## Step 1 : Pre-requisites for accessing the Vendasta's SCIM APIs
+
+### 1. Namespace
+
+This must be the id of the partner that wish to manage users in sync with Vendasta. It should be unique for a partner.
+
+### 2. Authorization token
+- For a partner with Vendasta, in order to access APIs of Vendasta they must need a authorisation token which is generated against the namespace of the partner. This should be refreshed over a particular time.
+
+- The authorization token must be generated with required scopes in order to access a particular platform.
+
+### 3. Scope
+- It is used to specify the access rights the partner should have towards Vendasta’s SCIM APIs.
+
+- user.admin: Read-write access to manage all users
+
+> To create a service account and create a token , see [here](../../Authorization/2-legged-oauth/Overview.md).
+
+
+
+## Step 2 : Vendasta SCIM Endpoints to sync users
+
+Endpoint Name | Method | URL | &nbsp;&nbsp;&nbsp; Description 
+---------|----------|---------|---------------
+ [Create User](../../../openapi/scim/scim.yaml/paths/~1{namespace}~1Users) | POST | /scim/{namespace}/Users | Create User based on SCIM schema | [Create API]((../../openapi/scim/scim.yaml))
```

Comment:
Add details about schema that we support. Add this page as a reference to schema https://developers.vendasta.com/platform/e7f8beaa13165-user